doc/en/_templates/links.html

```diff
@@ -6,6 +6,6 @@
   <li><a href="https://github.com/pytest-dev/pytest/">pytest @ GitHub</a></li>
   <li><a href="http://plugincompat.herokuapp.com/">3rd party plugins</a></li>
   <li><a href="https://github.com/pytest-dev/pytest/issues">Issue Tracker</a></li>
-  <li><a href="http://pytest.org/latest/pytest.pdf">PDF Documentation</a>
```

Comment:
perhaps the link should show to the exact own release on rtd
else older releases will point to the up2date pdf